dali/test/python/test_operator_warp.py

```diff
@@ -244,7 +261,7 @@ def test_gpu_vs_cpu():
                                             inv_map=inv_map)
                 gpu_pipeline.build()
 
-                compare(cpu_pipeline, gpu_pipeline, 1)
+                compare(cpu_pipeline, gpu_pipeline, 1.0001)
```

Comment:
Now the fill value is exactly half way (12.5), which results in different rounding on GPU (towards even == 12) vs CPU (away from zero == 13) - hence, larger epsilon.